elements/storytelling/src/style.eox.js

```diff
@@ -270,12 +270,21 @@ ${slider}
   .story-telling .hero {
     position: relative;      
     width: 100%;
-    height: 100dvh;
     overflow: hidden;
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
+    padding: 15rem 0;
+  }
+  @media screen and (max-width: 768px) {
+    .story-telling .hero {
+      padding: 10rem 0;
```

Comment:
Please add some horizontal padding, e.g. 1rem, as we noticed that some longer titles "stick" to the sides and it looks a bit odd. Thanks!